tests/chains_for_testBalance.json

```diff
@@ -318,5 +318,10 @@
         "chainId": "6fbd74e5e1d0a61d52ccfe9d4adaed16dd3a7caa37c6bc4d0c2fa12e8b2f4063",
         "name": "Polymesh Mainnet",
         "account": "0xf2aee43ecd134756088d19159a44f8733c36d8d5fd63fc9ffb8821b4ad148a7e"
+    },
+    {
+        "chainId": "b34f6cd03a41f0fab38ba9fd5b11cce5f303633c46f39f0c6fdc7c3c602bafa9",
+        "name": "Snow",
+        "account": " - "
```

Comment:
Should we add an account?

Reply:
yes, right, it's manual part of updating